File: week-03/day-3/07.py
# create a function that takes a list and returns a new list with all the elements doubled

list = [1, 2, 3, 4, 5, 33, 54, 67]

def double_list(list):
    new_list = []
    for e in list:
        new_list.append(e*2)
    return new_list

print(double_list(list))

File: week-07/day-03/07.js
'use strict';

var numbers = [2, 5, 11, 29];

// create a function that takes an array of numbers and returns a boolean
// it should return true if all the elements are prime, false otherwise

function prime (arr) {
  return arr.every(function(a){
    if (a >2) {
      for (var x = 2; x<a; x++) {
        return a % x !== 0
      }
    } else if (a === 2 || a === 1){
      return true
    } else {
      return false
    }
  })

}

console.log(prime(numbers))
